.ci/update_dpf_dependencies.py

```diff
@@ -45,24 +45,33 @@
                     file,
                     path=destination,
                 )
-    print("Done updating ansys-grpc-dpf")
+    print("Done updating ansys.grpc.dpf")
 else:
     print(f"{grpc_path_key} environment variable is not defined. " "Cannot update ansys-grpc-dpf.")
 
 if gate_path is not None:
     # Update ansys-dpf-gate
-    print("Updating ansys.dpf.gate")
-    dist_path = os.path.join(gate_path, "ansys-dpf-gate", "ansys")
+    print("Updating ansys.dpf.gate generated code")
```

Comment:
@rafacanton @BClappe the TFS CI uses this script here to update ansys-dpf-core based on ansys-dpf-gate, so the issue was that this change is not yet merged. I'll test it and keep you updated.

Reply:
Can confirm this now works on TFS